pkg/auth/providers/ldap/ldap_client.go

```diff
@@ -19,7 +19,7 @@ import (
 
 var operationalAttrList = []string{"1.1", "+", "*"}
 
-func (p *ldapProvider) loginUser(credential *v3public.BasicLogin, config *v3.LdapConfig, caPool *x509.CertPool) (v3.Principal, []v3.Principal, error) {
+func (p *ldapProvider) loginUser(credential *v3public.BasicLogin, config *v3.LdapConfig, caPool *x509.CertPool, enabled bool) (v3.Principal, []v3.Principal, error) {
```

Comment:
dont call the field enabled. call it `testServiceAccountBind` (which will affect the vaues you are passing from the calling functions)

Reply:
ok, get it